src/drag-and-drop/components/drop-zone-preview.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../shared/styles/helpers";
+
+$border-width: 4px;
+.dropZonePreview {
+  border: $border-width dotted #144dec;
```

Comment:
Do we define color constants somewhere in this project?

Reply:
No that I've seen

Reply:
No worries.  Just curious.